mantidimaging/gui/windows/main/view.py

```diff
@@ -757,3 +765,7 @@ def show_move_stack_dialog(self, origin_dataset_id: uuid.UUID, stack_id: uuid.UU
         self.move_stack_dialog = MoveStackDialog(self, origin_dataset_id, stack_id, origin_dataset_name,
                                                  stack_data_type)
         self.move_stack_dialog.show()
+
+    def show_stack_properties_dialog(self, stack_id: uuid.UUID, origin_dataset: Dataset, stack_data_type: str) -> None:
+        self.stack_properties_dialog = StackPropertiesDialog(self, stack_id, origin_dataset, stack_data_type)
+        self.stack_properties_dialog.show()
```

Comment:
Can probably just be
```
stack_properties_dialog = StackPropertiesDialog(self, stack_id, origin_dataset, stack_data_type)
stack_properties_dialog.show()
```
Because we done need to access it again from the main window. Also as it currently is, the value gets over written if multiple dialogs are opened.